python/tests/test_lowlevel.py

```diff
@@ -138,6 +138,11 @@ def get_example_migration_tree_sequence(self):
         )
         return ts.ll_tree_sequence
 
+    def get_empty_tree_sequence(self):
+        tables = tskit.TableCollection(sequence_length=2)
+        ts = tables.tree_sequence()
+        return ts.ll_tree_sequence
+
```

Comment:
This looks unused?

Reply:
Ah, thanks for catching this! I neglected to remove this after reducing the low level tests.